spark/dl/src/main/scala/com/intel/analytics/bigdl/models/resnet/README.md

```diff
@@ -73,3 +74,56 @@ We support Local and Spark versions of training. Users can define <code>env</cod
     --dampening     [default 0.0]
     --nesterov      [default true]
 ```
+## Train ResNet on ImageNet
```

Comment:
should we post the hyperparameters and training result for reference?

Reply:
ok, just added separate section